pkg/txm/clientwrappers/dualbroadcast/flashbots_client.go

```diff
@@ -20,25 +20,25 @@ import (
 	"github.com/smartcontractkit/chainlink-evm/pkg/txm/types"
 )
 
```

Comment:
Might be good to add a type assertion, e.g.:

```suggestion
var _ txm.Client = (*FlashbotsClient)(nil)
```